schema/me.js

```diff
@@ -40,10 +37,39 @@ const Me = new GraphQLObjectType({
       resolve: (root, { current }, { rootValue: { accessToken } }) => {
         return gravity.with(accessToken)('me/bidder_positions')
           .then((positions) => {
-            const withHighestBid = reject(positions, { highest_bid: null });
-            return current
-              ? uniqBy(withHighestBid, 'sale_artwork_id')
-              : positions;
+            if (!current) return positions;
+
+            // When asking for "my current bids" we need to...
+            //
+            // 1. Find only positions that are "last placed" and
+            // "competing to win" for that user, which means finding the most
+            // recently created bidder positions per sale artwork where
+            // `position.highest_bid != null`.
+            //
+            const latestPositions = _(positions).chain()
+              .reject({ highest_bid: null })
+              .uniqBy('sale_artwork_id')
+              .value();
+            //
+            // 2. Find only bidder positions in "open" auctions. This requires
+            // fetching all of that related data to be able to do:
+            // `bidder_position.sale_artwork.sale.auction_state != open`
```

Comment:
After testing "Your Active Bids" through a couple staging auctions I realized it would return bidder positions for auctions that already closed, which is not the desired behavior, hence the :spaghetti: of getting the whole `sale`, `sale_artwork`, `bidder_position` family together.